### ISSUE TITLE
Optimisation: Replacing listView in CarSelectionActivity with Recycler View having well designed Cardviews

### ISSUE THREAD

**Issue by swayamkaul** [enviroCar/enviroCar-app#674]
![Screenshot 2021-03-27 at 2 26 30 AM](https://user-images.githubusercontent.com/43716674/112692174-8c074700-8ea4-11eb-8851-4d3c60c61fa0.png)
This activity presently contains listView for showing the list of cars which is not an efficient approach.
This listView needs to be replaced with RecyclerView having well designed CardViews which will lead to a better UserInterface as well as better app performance.

**Comment by swayamkaul** [comment]
**I will be working on this issue**

**Comment by DhirajChauhan40** [comment]
its been more than 4 months if @swayamkaul you have not done yet can i work on it? @asaikarthikeya

**Comment by asaikarthikeya** [comment]
@DhirajChauhan40 Yes, you can work on this. Could you provide a mock up design on how you would like to improve the design ?

**Comment by DhirajChauhan40** [comment]
yes @asaikarthikeya  I have something  which i used in my hotel app project, I can show you the screenshots

**Comment by asaikarthikeya** [comment]
> yes @asaikarthikeya I have a design which i used in my hotel app project, its pretty subtle and at the same time looks good I can show you the screenshots

You can use that design and add the items and views of this layout. You can share the mock up after adapting the change

**Comment by DhirajChauhan40** [comment]
![Screenshot_2021-08-23-16-16-59-70_0a7da8b3cac176690a1be5c50006612f.jpg](https://user-images.githubusercontent.com/75211982/130435116-a3bd69e2-dcb8-4324-a13a-21ce264bb721.jpg)
here at the photo of bed we can replace with the logo of the company or the photo of car itself(but i don't know how to do with every car) , and other details

**Comment by DhirajChauhan40** [comment]
update: I'm just gonna take idea from here, the UI wont be that much similar to the above design

**Comment by DhirajChauhan40** [comment]
@asaikarthikeya I'm having difficulty with connecting all the files and data which is from other different java files, can you help me?

**Comment by DhirajChauhan40** [comment]
> @DhirajChauhan40 Sure. Actually, This issue is not a beginner friendly one, and the design aspect involved in this issue needs some internal discussion before finalizing and working on. I would suggest you to work on a few relatively easy issue which make you familiar with the code base of enviroCar. Then you can take on the challenging onces. Feel free to ask doubts. I will try to answer them in my free time :) 

oh sure actually today i had talk with Benjamin and he also said the same to me
so do you have some issues in your mind which you can state here
will be really helpful! @asaikarthikeya

**Comment by DhirajChauhan40** [comment]
> > @DhirajChauhan40 Sure. Actually, This issue is not a beginner friendly one, and the design aspect involved in this issue needs some internal discussion before finalizing and working on. I would suggest you to work on a few relatively easy issue which make you familiar with the code base of enviroCar. Then you can take on the challenging onces. Feel free to ask doubts. I will try to answer them in my free time :) 
> 
> oh sure actually today i had talk with Benjamin(mentor) and he also said the same to me
> so do you have some issues in your mind which you can state here
> will be really helpful! @asaikarthikeya

**Comment by asaikarthikeya** [comment]
@DhirajChauhan40 I will find create/ find a issue and tag you there.

**Comment by DhirajChauhan40** [comment]
> @DhirajChauhan40 I will find create/ find a issue and tag you there. 

thank you so much!!

**Comment by MohitGupta121** [comment]
Is this issue is still open, I want to work on this issue @cdhiraj40 ?

**Comment by asaikarthikeya** [comment]
> Is this issue is still open, I want to work on this issue @cdhiraj40 ?

Yes, It is open. please refer to the above conversation. Thanks

**Comment by MohitGupta121** [comment]
> 

Okay thanks you. I'll check some other issues.
